docker-compose-dev.yml

```diff
@@ -22,7 +22,7 @@ services:
     command: bin/rails server -b 0.0.0.0
     environment:
       DB_USERNAME: postgres
-      DB_PASSWORD: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
The ENV variables to read db credentials changed to POSTGRES_PASSWORD but this was not updated